tests/Task1/test1.performLogin.spec.js

```diff
@@ -0,0 +1,32 @@
+const { test, expect } = require('@playwright/test');
+const selectors = require('../../page-objects/selectors');
+const users = require('../../page-objects/users');
+
+test('Verify login functionality and UI elements on the home page', async ({ page }) => {
+  // Navigate to the Sauce Demo login page
```

Comment:
You can add test.step() for a group of commands, instead of leaving just a comment. These step you can see in the HTML report - https://playwright.dev/docs/api/class-test#test-step

Reply:
done